src/Ydb.Sdk/src/Driver.cs

```diff
@@ -232,8 +222,7 @@ private async Task<Status> DiscoverEndpoints()
 
     private async Task PeriodicDiscovery()
     {
-        var stop = false;
-        while (!stop)
+        while (!_disposed)
```

Comment:
with old code the loop was run minimum one time.
with new version - it will no run if _disposed already true